pkg/lokifrontend/frontend/v1/frontend.go

```diff
@@ -327,7 +327,7 @@ func (f *Frontend) queueRequest(ctx context.Context, req *request) error {
 	joinedTenantID := tenant.JoinTenantIDs(tenantIDs)
 	f.activeUsers.UpdateUserTimestamp(joinedTenantID, now)
 
-	err = f.requestQueue.Enqueue(joinedTenantID, req, maxQueriers, nil)
+	err = f.requestQueue.Enqueue(joinedTenantID, nil, req, maxQueriers, nil)
```

Comment:
Do we want to enable hierarchical queuing in the v1 frontend (single binary!) as well? If so, we probably need to change the `max_queue_hierarchy_levels` setting from being a scheduler setting to being a frontend setting (which is not great).

Reply:
It's say no.